package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "botframework-webchat-root",
-  "version": "4.9.2-0",
+  "version": "4.9.2",
```

Comment:
Please wait until 4.9.2 is released on CDN. Then we do post-release 4.9.2.

This version number will become `4.9.3-0`.